src/Driver/PDO/Connection.php

```diff
@@ -123,8 +123,23 @@ public function rollBack(): bool
         return $this->connection->rollBack();
     }
 
+    public function getNativeConnection(): PDO
+    {
+        return $this->connection;
+    }
+
+    /**
+     * @deprecated Call {@see getNativeConnection()} instead.
+     */
     public function getWrappedConnection(): PDO
     {
+        Deprecation::triggerIfCalledFromOutside(
+            'doctrine/dbal',
+            'TODO',
+            '%s is deprecated, call getNativeConnection() instead.',
+            __METHOD__
+        );
+
         return $this->connection;
```

Comment:
Should it call the new method? This way, it will illustrate the upgrade path and provide better code coverage.

Reply:
Done.